Wiki/starting/starting-leaving.md

```diff
@@ -0,0 +1,146 @@
+* What files will be useful to others? Should they be published?
+* What files won't be useful to others? Should they be archived or deleted?
+
+```{note}
+If you need specific advice on how to actually transfer or archive your files, you can always ask for assistance to our helpdesk: cws_help@nci.org.au.
+```
+
+## What to do
+
+While what exactly should be kept, removed or published will depend on your specific project and working group. There are some principles that can be applied to any situation.
+ 
+### To keep, always
+
+**Codes**
+
+No matter what the details of your projects are you should always keep the code you used. This means keeping a reference to the exact version and document how you used codes developed by others (e.g. climate models). If you modified them in any way this should also be documented. 
+
+If you modfication are extensive or you have written the code yourself then you should make it available via GitHub and publish it (Zenodo is the most indicated for this).
+
+```{note}
+It is absolutely fine to create a repository per project or paper with all your codes in. You can then use the README file from the GitHub repository to explain how to reproduce your results. Don't forget to clearly reference everything one might need in addition to this repository. Or you can have a repository per code especially if you envision you'll reuse the same code for other work.
+```
+
+**Configuration files for running the codes and some input files**
+
+In addition to the codes themselves, you need to keep everything that enables someone to run the codes in the same way you have done so. Usually, the most complicated configurations are for climate models. Some climate models will save your configurations in version control repositories (e.g. UM, ACCESS, ACCESS-OM2), in which case you simply need to keep the information on how to retrieve these configurations. Some models don't save your configurations and you need to do it yourself.
+
+For the input files, some inputs are published data in which case you need to keep the reference to this data (including the version). If you have written several codes, the output of a piece of code will be the input of the next piece of code, in which case you do not necessarily need to keep that data,  but you need to describe your workflow.
+
+**Workflow**
+
+This can be a tricky one, as there is not a one-size-fits-all format to save this information. It is fine to write a README file and archive it with other files from the project that need archiving. You can also have a special Github repository just for this README file, or a repository for all your projects with READMEs for each project. Whatever format you choose, it is important for this information to be publicly available (unless your project was restricted) and not a personal note.
+
+This description should clearly describe step by step what someone should do to reproduce your work.
```

Comment:
```suggestion

```

Reply:
If I approve this it will show "I have appended this line to the previous paragraph" rather than the intended change

Reply:
Sorry

Reply:
done